core/services/offchainreporting/validate.go

```diff
@@ -44,17 +44,10 @@ func ValidatedOracleSpecToml(config ValidationConfig, tomlString string) (job.Jo
 	}
 	jb.OffchainreportingOracleSpec = &spec
 
-	// TODO(#175801426): upstream a way to check for undecoded keys in go-toml
```

Comment:
There is a way, see large comment in job/validate.go. Hinges on schemaVersion debate outcome